qdrant_client/async_qdrant_fastembed.py

```diff
@@ -125,15 +142,62 @@ def _scored_points_to_query_responses(
             )
         return response
 
+    def _records_iterator(
+        self,
+        ids: Optional[Iterable[models.ExtendedPointId]],
+        metadata: Optional[Iterable[Dict[str, Any]]],
+        encoded_docs: Iterable[Tuple[str, List[float]]],
+        ids_accumulator: list,
+    ) -> Iterable[models.Record]:
+        if ids is None:
+            ids = iter(lambda: uuid.uuid4().hex, None)
+        if metadata is None:
+            metadata = iter(lambda: {}, None)
+        vector_name = self.get_vector_field_name()
+        for idx, meta, (doc, vector) in zip(ids, metadata, encoded_docs):
+            ids_accumulator.append(idx)
+            payload = {"document": doc, **meta}
```

Comment:
if you get picky, here we can actually overwrite "document" if `meta` has such a field

Reply:
yes, I also don't like this, but we can't change it anymore

Reply:
somehow missed it in last release